src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,71 @@
+import { useMemo, useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { getFilteredPeople } from '../utils/getFilteredPeople';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
if you have a boolean value its better to start a name with `is` or `has` like `isLoading`